File: api/app/InterApi.Entities/ApiModels/AccountActivity.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InterApi.Entities.ApiModels
{
    public class AccountActivity
    {
        public string Date { get; set; }
        public string DueDate { get; set; }
        public string Explanation { get; set; }
        public long Amount { get; set; }
        public long RemainingBalance { get; set; }
        public long TransactionReference { get; set; }
        public string TransactionCode { get; set; }
        public long TransactionBranchCode { get; set; }
        public string ChannelCode { get; set; }
        public string Time { get; set; }
        public string UserCode { get; set; }
        public string GroupRef { get; set; }
        public long AvailableBalance { get; set; }
        public string ReferenceNumber { get; set; }
    }
}

File: ui/app/src/app/models/accountActivity.ts
export interface AccountActivity {
        date: string;
        dueDate?: string;
        explanation: string;
        amount: number;
        remainingBalance: number;
        transactionReference: number;
        transactionCode?: string;
        transactionBranchCode: number;
        channelCode?: string;
        time?: string;
        userCode?: string;
        groupRef?: string;
        availableBalance: number;
        referenceNumber?: string;
}
